src/measure_theory/borel_space.lean

```diff
@@ -221,6 +221,24 @@ instance opens_measurable_space.to_measurable_singleton_class [t1_space α] :
   measurable_singleton_class α :=
 ⟨λ x, is_closed_singleton.is_measurable⟩
 
+instance pi.opens_measurable_space {ι : Type*} {π : ι → Type*} [fintype ι]
+  [t' : Π i, topological_space (π i)]
+  [Π i, measurable_space (π i)] [∀ i, second_countable_topology (π i)]
+  [∀ i, opens_measurable_space (π i)] :
+  opens_measurable_space (Π i, π i) :=
+begin
+  refine ⟨_⟩,
```

Comment:
Why not putting those brackets outside the tactic block, or use the `constructor` tactic?

Reply:
I always forget the name `constructor`.